web/pgadmin/tools/sqleditor/static/js/components/sections/Query.jsx

```diff
@@ -417,12 +350,64 @@ export default function Query({onTextSelect, setQtStatePartial}) {
       }
     };
 
+    const unregisterEditorExecCmd = eventBus.registerListener(QUERY_TOOL_EVENTS.EDITOR_EXEC_CMD, (cmd='')=>{
+      if(cmd == 'gotoLineCol') {
```

Comment:
Will gotoLineCol work outside query tool? If not we should add a cmd for this as well.

Reply:
It works outside query tool.